ice/src/util/mod.rs

```diff
@@ -118,17 +118,13 @@ pub async fn local_interfaces(
 
             if !ipaddr.is_loopback()
                 && ((ipv4requested && ipaddr.is_ipv4()) || (ipv6requested && ipaddr.is_ipv6()))
+                && ip_filter
```

Comment:
Why is this diff here, this is already the code on `master`, something must've gone wrong with rebasing